include/avif/avif.h

```diff
@@ -305,8 +305,12 @@ typedef enum avifChromaSamplePosition
 
 typedef enum avifRange
 {
-    AVIF_RANGE_LIMITED = 0,
-    AVIF_RANGE_FULL = 1
+    AVIF_RANGE_LIMITED = 0, /**<- Y  [16..235],  UV  [16..240]  (bit depth 8) */
+                            /**<- Y  [64..940],  UV  [64..960]  (bit depth 10) */
+                            /**<- Y [256..3760], UV [256..3840] (bit depth 12) */
+    AVIF_RANGE_FULL = 1     /**<- YUV/RGB [0..255]  (bit depth 8) */
+                            /**<- YUV/RGB [0..1023] (bit depth 10) */
+                            /**<- YUV/RGB [0..4095] (bit depth 12) */
```

Comment:
I think it is better to remove "/RGB" from this comment. I think "/RGB" means that R, G, B are always in full range, but this is not obvious from reading the comment.

Note: There is a special case for limited range: Identity (MatrixCoefficients=0) and YCgCo (MatrixCoefficients=8). In this special case, the limited range is the same for all three channels (G,B,R, or Y,Cg,Co).

Reply:
Perhaps we can write the comments as follows:

```
    AVIF_RANGE_LIMITED = 0, /**<- Y  [16..235],  UV  [16..240]  (bit depth 8) */
                            /**<- Y  [64..940],  UV  [64..960]  (bit depth 10) */
                            /**<- Y [256..3760], UV [256..3840] (bit depth 12) */
    AVIF_RANGE_FULL = 1     /**<- [0..255]  (bit depth 8) */
                            /**<- [0..1023] (bit depth 10) */
                            /**<- [0..4095] (bit depth 12) */
```

And add another comment that says "avifRange is not applicable to RGB and alpha planes." or "avifRange is only applicable to YUV planes. RGB and alpha planes are always full range."